### ISSUE TITLE
`[2022-03-11 20:14:31][datar][   INFO] Adding missing grouping variables: ['efficiency']` warning message

### ISSUE THREAD

**Issue by rleyvasal** [pwwang/datar#89]
what the following warning mesage about?

`[2022-03-11 20:14:31][datar][   INFO] Adding missing grouping variables: ['efficiency']`

I am using a `for` loop to iterate over thousands of columns and the warning message keeps showing in output.  I created some code below to illustrate the warning message. 

## Code to replicate
```python
mtcars_vars = mtcars.columns.to_list()
 mtcars = mtcars >> mutate(efficiency = case_when(
                    f.mpg.between(0,16, inclusive ='left'), "bad",
                    f.mpg.between(16,20, inclusive ='left'), "ok",
                    f.mpg.between(20,22, inclusive ='left'), "better",
                    f.mpg>=22, "best",
                    True, "other" )      

col_name = mtcars_vars[1]
all_cars = mtcars >> select(f.efficiency, f[col_name]) >> filter(f[col_name] > 0) >> group_by(f.efficiency) >> count() >> rename(**{col_name: f.n})
```

**Comment by pwwang** [comment]
This is a false alarm.

This happens when grouping variables are not selected:

```python
>>> df = tibble(x=1, y=2) >> group_by(f.x)
>>> df >> select(f.y)
[2022-03-12 13:52:33][datar][   INFO] Adding missing grouping variables: ['x']
        x       y
  <int64> <int64>
0       1       2
[TibbleGrouped: x (n=1)]
```

In `rename()`, `col_name` was passed to `select` (backend), without the grouping variable `efficiency`. That's why there is a such warning. But `rename()` is not subsetting the data frame (the grouping variables are anyway included), so the warning should not exist.

Nice hit!

**Comment by pwwang** [comment]
Fixed at [v0.6.2](https://github.com/pwwang/datar/releases/tag/0.6.2)
